MapleServer2/Database/Classes/DatabaseItem.cs

```diff
@@ -59,15 +59,28 @@ public long Insert(Item item)
             transfer_flag = item.TransferFlag,
             blackmarket_category = item.BlackMarketCategory,
             transparency_badge_bools = JsonConvert.SerializeObject(item.TransparencyBadgeBools),
-            unlock_time = item.UnlockTime
+            unlock_time = item.UnlockTime,
+            ugc_uid = item.UGC == null ? null : (int?) item.UGC.Uid,
```

Comment:
this will bring 0 if is null ? have doubts on this sometimes

Reply:
Null because 0 is a valid number so it'll try to find the reference in the other table but of course it doesn't exists